src/util.ts

```diff
@@ -51,14 +51,13 @@ export function errToString(error: any): string {
 
 export function showResult(result: string, filename: string, column?: vscode.ViewColumn): Thenable<void> {
 	let uri: vscode.Uri = null;
-	if (vscode.workspace.rootPath) {
-		uri = vscode.Uri.file(path.join(vscode.workspace.rootPath, filename));
+	const currExtensionPath = vscode.extensions.getExtension("ms-azuretools.vscode-cosmosdb").extensionPath;
```

Comment:
I'm not too comfortable about having the extension ID `ms-azuretools.vscode-cosmosdb` as a string. Ideally, I should be pulling this from the package.json file (publisher and name fields).

Reply:
You can pass context.extensionPath all the way from the activate() method in extension.ts. That way you don't even have to 'getExtension'

Something like: 
```
MongoCommands.executeCommandFromActiveEditor(connectedDb, context.extensionPath)
```
```
await util.showResult(result, 'result.json', activeEditor.viewColumn + 1, extensionPath);
```